src/datastore/factory.jl

```diff
@@ -2,15 +2,19 @@
 using Pkg
 datastore_env() = get(ENV, "DATASTORE", "")
 
-global DATASTORE_MODULE = let
+const DATASTORE_MODULE = Ref{Module}()
+
+begin
```

Comment:
Why is there a begin block, is it necessary?

Reply:
Not necessary but logically combines the code